src/libguac/guacamole/wol-constants.h

```diff
@@ -39,13 +39,5 @@
  */
 #define GUAC_WOL_PACKET_SIZE 102
 
-/**
- * The port number that the magic packet should contain as the destination.  In
- * reality this doesn't matter all that much, since the packet is not usually
- * processed by a full IP stack, but defining one is considered a standard
- * practice.
- */
-#define GUAC_WOL_PORT 9
-
```

Comment:
Why remove the port constant from libguac?

Reply:
It gets moved over to the `defaults.h` file as a default value, since we're now allowing it to be specified. UDP port 9 is officially the "Discard Protocol", and has been unofficially adopted for Wake-on-LAN. So, it technically isn't the "Wake-on-LAN" port, just the one that gets used most times.

I'm happy to back this change out and just leave it as is, using this value as the default, I was just recalling the discussion we had about defaults when originally implementing this support and trying to put it in the proper place.

Reply:
So long as the port is a standard, I think this is still the correct place. If this were not a standard port, and we just chose that number because we like it as a default for our own protocol plugins, then the internal `defaults.h` would make more sense.

Reply:
Okay, sounds good - I've moved it back and confirmed that everything builds correctly.